stub-images/virt-v2v-stub/bin/entrypoint

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+qemu-img info /mnt/disks/disk0/disk.img
+qemu-img convert -f raw -O qcow2 /mnt/disks/disk0/disk.img /mnt/disks/disk0/disk.qcow2
```

Comment:
Thinking about it more, I wonder what would be the benefit of doing this.

Reply:
yeah, in theory we can have a no-op stub but then the test could succeed even if we don't wait for the conversion pod to finish - that may be an overkill but if it converts the image from other format to raw (not as it is now - kubevirt doesn't support qcow2 yet) then we can be sure at the end that if we get a valid raw disk (or able to start the vm), the migration worked as expected

Reply:
Let's keep it there.